frontend/packages/substrate-react/src/dotsama/hooks/useEagerConnect.ts

```diff
@@ -36,7 +36,10 @@ export const useEagerConnect = (chainId: ParachainId): boolean => {
 
   useEffect(() => {
     if (selectedAccount) {
-      localStorage.setItem("selectedAccount", selectedAccount.address);
+      localStorage.setItem(
+        `selectedAccount-${chainId}`,
```

Comment:
@easteregg what's the intention of storing chain id as well?

Reply:
Eager connect failes to connect when we pass in other chains as well, mainly it fails to find the right wallet, so we need to have those values as well.